Assets/AirshipPackages/@Easy/Core/Shared/MainMenu/Components/AirshipScreenSafeArea.ts

```diff
@@ -25,6 +40,32 @@ export default class AirshipScreenSafeArea extends AirshipBehaviour {
 
 	protected UpdateSafeArea() {
 		if (!Game.IsMobile()) return;
+
+		let platform = Game.platform;
+		if (Game.IsEditor()) {
+			platform = AirshipPlatform.Android;
```

Comment:
should remove this and use whatever we infer from Game.platform right? IIRC Game.platform takes the simulator into account.